.gitignore

```diff
@@ -40,3 +40,4 @@ yarn-debug.log*
 *.swp
 .DS_Store
 .env*
+.env*
```

Comment:
You don't need the '*' and the line is now duplicated 😅

It can be like this:
.env

Reply:
Thanks!